### DISCUSSION TITLE
Hide Callout on Android

### DISCUSSION THREAD

**Discussion by gunnar-bell** [react-native-maps/react-native-maps#4691] [Q&A]
### Summary

I cannot figure out a way to hide the callout on the Android platform. I was able to successfully hide it on iOS though. Is there a way to remove it entirely on Android?

### Reproducible sample code

```TSX
import React from 'react'
import MapView, { Marker } from 'react-native-maps'

<MapView ...props>
  {markers.map(marker => (
    <Marker coordinate={marker.coordinate}>
      <Callout tooltip />
    </Marker>
  ))}
</MapView
```

### Steps to reproduce

1. Be on Android
2. Render a map with markers
3. Have an empty callout with the `tooltip` prop
4. Tap on a marker
5. Observe that the callout shows

### Expected result

The callout does not show

### Actual result

the callout shows

### React Native Maps Version

1.3.2

### What platforms are you seeing the problem on?

Android

### React Native Version

0.70.5

### What version of Expo are you using?

SDK 47

### Device(s)

All Android

**Comment by bvitlas** [comment]
same here. did you solve it maybe ?

**Comment by rakumairu** [comment]
This works for me on android
```
<Callout
    tooltip
    style={{ display: 'none' }}
>
    <View>
        <Text>
            Hidden
        </Text>
    </View>
</Callout>
```

**Comment by fdesjardins** [comment]
I was still having this issue on Android recently and this is the workaround I was most satisfied with:
```jsx
  // If you try to conditionally render the callout or conditionally 
  // set the style you'll either get errors or nothing will happen. 
  // However, if you change the marker key, the marker gets 
  // re-rendered which has the side effect of closing the callout.
  let key = `${markerId}-${calloutHidden}`;

  return (
    <Marker
      key={key}
```

I suspect the issue has something to do with the `collapsable` prop on the `Marker` and/or `Callout`, but I moved on once I had the workaround above.
